### ISSUE TITLE
Flashing cursor in the margin and poor performance in terminal with linum

### ISSUE THREAD

**Issue by morgado-jb** [syohex/emacs-git-gutter#106]
There is a noticeable performance penalty when using git-gutter in terminal mode and scrolling.

The cursor keeps flashing in the top left side of the screen (in the margin at the 1st line on the left) and I think it's what causes this that is slowing things down.

**Comment by syohex** [comment]
Yes git-gutter.el with linum makes too slow. Because linum updates line-number very frequently. If you want to such tools with linum, I recommend you to use [diff-hl](https://github.com/dgutov/diff-hl) or [git-gutter-fringe](https://github.com/syohex/emacs-git-gutter-fringe).

**Comment by gonewest818** [comment]
Is this still the case in 2020?  I'm not seeing it in Emacs 27.0.90 in `-nw` mode.
